### PR TITLE
Updated page with necessary Color Picker tool link

### PR DESCRIPTION
**Pull request by kostasx**
### Description

The page did not include a direct link to the Color picker tool. A reference to the link was added to the page in order for the visitor to quickly launch the tool.

### Motivation

To help readers use the tool mentioned by the page.

### CODE REVIEW

File: files/en-us/web/css/css_colors/color_picker_tool/index.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ page-type: guide
 
 {{CSSRef}}
 
-This tool makes it easy to create, adjust, and experiment with custom colors for the web. It also makes it easy to convert between various [color formats](/en-US/docs/Web/CSS/color_value) supported by [CSS](/en-US/docs/Web/CSS), including: HEXA colors, RGB (Red/Green/Blue) and HSL (Hue/Saturation/Lightness). Control over the alpha channel is also supported.
+[This tool](https://mdn.github.io/css-examples/tools/color-picker/) makes it easy to create, adjust, and experiment with custom colors for the web. It also makes it easy to convert between various [color formats](/en-US/docs/Web/CSS/color_value) supported by [CSS](/en-US/docs/Web/CSS), including: HEXA colors, RGB (Red/Green/Blue) and HSL (Hue/Saturation/Lightness). Control over the alpha channel is also supported.
```
Comment (hamishwillee) [inline files/en-us/web/css/css_colors/color_picker_tool/index.md:9]:
```suggestion
The Color Picker tool makes it easy to create, adjust, and experiment with custom colors for the web. It also makes it easy to convert between various [color formats](/en-US/docs/Web/CSS/color_value) supported by [CSS](/en-US/docs/Web/CSS), including: HEXA colors, RGB (Red/Green/Blue) and HSL (Hue/Saturation/Lightness). Control over the alpha channel is also supported.
```

**Comment by hamishwillee** [comment]
@kostasx I'm closing this because the link is not necessary. If you look at the page, it refers to "This tool" because the color picker is embedded in the page. I.e. this page is the color picker tool, not the MDN example page.

**Comment by kostasx** [comment]
Oh, my God! You are absolutely right. False alarm.
Thank you @hamishwillee and sorry for wasting your time on this one.

**Comment by hamishwillee** [comment]
No worries at all - appreciate you taking the time to post a fix, even if it wasn't necessary in this particular case.

**Pull request closed without merging**